spec/03-plugins/26-oauth2/04-invalidations_spec.lua

```diff
@@ -12,15 +12,16 @@ for _, strategy in helpers.each_strategy() do
 
     setup(function()
       bp, db, dao = helpers.get_db_utils(strategy)
-
-      assert(db:truncate())
-      dao:truncate_tables()
-      assert(dao:run_migrations())
     end)
 
     before_each(function()
-      assert(db:truncate())
-      dao:truncate_tables()
+      assert(db:truncate("routes"))
+      assert(db:truncate("services"))
+      assert(db:truncate("consumers"))
+      dao:truncate_table("plugins")
+      dao:truncate_table("oauth2_tokens")
+      dao:truncate_table("oauth2_credentials")
+      dao:truncate_table("oauth2_authorization_codes")
```

Comment:
Did this commit use to have all of the tests changes, or were they supposed to be part of the subsequent commit (reduce unnecessary truncations)? Either if fine if it was the goal, just making sure, because I seem to recall those spec changes were not in this commit

Reply:
The spec changes related to the `truncate` calls were in this commit, and the spec changes related to the other modifications I made to the helpers ware in the other commit. But I ended up applying more and more truncation cleanups so they might have mixed up a bit, so I just moved all spec changes to the `tests(*)` commit now.

Reply:
Alright!